lockfile/src/lib.rs

```diff
@@ -48,6 +50,7 @@ pub enum LockfileFormat {
     Msbuild,
     Go,
     Cargo,
+    Sbom,
```

Comment:
After looking into this, I think this should be `Spdx` and not `Sbom`.

If I understand the terms correctly, SPDX is the file format that is being used here. That's why the files are named with `.spdx.json` (or `.yaml` or `.yml`) extensions. Our parser then uses the SBOM contained in that file to list packages.

But, critically, there are other file formats that may contain the SBOM (i.e., CycloneDX files). And if we decide to support those files in the future, they would use a separate parser.

I know this comment comes a bit late in the process.... But I'd like to avoid another `msbuild`/`nuget` situation. So I think we need to deal with this now.

Reply:
Sounds good, I'll update. I thought we'd handle all formats, starting with SPDX and others to follow, under `Sbom`, and add any additional [parsers](https://github.com/phylum-dev/cli/pull/978) that might be needed.